src/ljs_input.cpp

```diff
@@ -103,11 +103,15 @@ int main(int argc, char **argv)
     msg.bump_r_dwn = GamepadButtonDown(controller, BUTTON_RIGHT_SHOULDER);
 
     if(twist_lock==true and (GamepadTriggerLength(controller, TRIGGER_LEFT)<0.1)){
-    msg.trig_l_val = 0.435;
+    msg.trig_l_val = 0.0;
     }
     else{
-    msg.trig_l_val = GamepadTriggerLength(controller, TRIGGER_LEFT);
-    twist_lock = false;
+      msg.trig_l_val = GamepadTriggerLength(controller, TRIGGER_LEFT) - 0.435;
```

Comment:
Can we define these 0.435 vals as some constant like  and maybe move the explanation as to why we need this offcentre on (it's given on line 128 of rjs_input.cpp) to the constant definition?

Reply:
Yeah fair assessment, should also bring it to more than 3 decimal places too since I can quite easily find the actual centered value.